packages/flutter_tools/lib/src/ios/code_signing.dart

```diff
@@ -14,34 +14,57 @@ import '../base/process.dart';
 import '../base/terminal.dart';
 import '../globals.dart';
 
+/// When there are no development certificates at all. The user likely never
+/// did any iOS development.
```

Comment:
Consider dropping the second sentence; it doesn't add much. Maybe just:
```
/// Instructions when no development certificates are found in the keychain.
```

If not, re-org into a summary sentence, paragraph break, then details in a follow-on paragraph.

https://www.dartlang.org/guides/language/effective-dart/documentation#do-make-the-first-sentence-a-standalone-paragraph